boolean-normal-forms.cabal

```diff
@@ -42,7 +42,8 @@ library
                        ScopedTypeVariables
   build-depends:       base        >=4.6   && <4.11,
                        containers  >=0.5   && <0.6,
-                       cond        >=0.4.1 && <0.5
+                       cond        >=0.4.1 && <0.5,
+                       deepseq     >= 1.1.0.0
```

Comment:
`deepseq` is unluckily to change soon, but if it will have major bump, then it way or another will break most of its users. So add `<1.5` bound. Also if you mind, format as above (no space between comparison operator and version).